templates/locations/locations.html

```diff
@@ -121,10 +122,29 @@
 
 <script>
   
-   $(document).ready(function() {
+   function init() {
       // Activate the top navbar
       $("#navid-locations").addClass("active");
-   });
+      
+      // Show correct selection in kind popup button
+      var kind_name = 'Allir staðir';
```

Comment:
Change naming convention to camelCase for JavaScript (kindName instead of kind_name)